README.md

```diff
@@ -39,7 +39,7 @@ jobs:
     - id: build-publish
       uses: bitovi/github-actions-react-to-ghp@v1.2.0
       with:
-        path: dist
+        path: build # change to your build folder
```

Comment:
> I found in react the default folder was build, and default branch to be master when creating a new git project.
As this is the readme/docs, it's meant as an example as people might use many different build systems that output in all sorts of different places.

I'm good with this as-is, and I especially love the comment next to the example <3